src/components/PeoplePage/PeopleTable/PersonRow/PersonRow.tsx

```diff
@@ -0,0 +1,36 @@
+import { NewPerson } from '../../../../react-app-env';
+
+interface Props {
+  person: NewPerson,
+}
+
+export const PersonRow: React.FC<Props> = ({ person }) => (
+  <tr>
+    <td>{person.name}</td>
+    <td>{person.sex}</td>
+    <td>{person.born}</td>
+    <td>{person.died}</td>
+    <td>
+      {person.mother !== null ? (
+        <ul className="m-0" style={{ listStyleType: 'none' }}>
+          <li>{person.mother.name}</li>
+          <li>{`Born: ${person.mother.born}`}</li>
+          <li>{`Died: ${person.mother.died}`}</li>
+        </ul>
```

Comment:
This element is the same for mother and father, you can create separate component for this